src/modules/Client/Api/Admin.php

```diff
@@ -256,7 +256,10 @@ public function update($data = [])
                 throw new \Box_Exception('Can not change email. It is already registered.');
             }
         }
-        $this->di['validator']->isBirthdayValid($data['birthday'] ?? null);
+
+        if (!is_null($data['birthday'] ?? null)) {
+            $this->di['validator']->isBirthdayValid($data['birthday'] ?? null);
```

Comment:
This doesn't need a null coalescing operator and can just be `$data['birthday']`.

Reply:
Right! I thought so too. But, I just took the if from check for email in the above line and pasted it for birthday.

Reply:
The `is_null` if statement still needs to have it, as `is_null` is only checking if a variable is set to null and it will still throw an error if the variable is not set.
Optionally, this can be done to remove the null coalescing operator entirely:
```PHP
if (!empty($data['birthday'])) {
    $this->di['validator']->isBirthdayValid($data['birthday']);
}
```

`empty` checks if the variable is set, null, or falsey and as such it won't cause an error if the variable is undefined

Reply:
oh right. I assumed that an empty string might work. I will update my code and re-commit.